exercises/say/example.go

```diff
@@ -8,7 +8,14 @@ var tens = []string{"ones", "ten", "twenty", "thirty", "forty",
 var scale = []string{"thousand", "million", "billion",
 	"trillion", "quadrillion", "quintillion"}
 
-func Say(n uint64) string {
+func Say(n int64) (value string, ok bool) {
```

Comment:
I'm good with ```ok bool``` here.  The standard library has instances like it.

Reply:
Does anybody know if/where there is some sort of community accepted definition or guideline as for when to use an `err` vs an `ok`?

Reply:
I have not found a straight out "accepted definition".  There is discussion. Here is one from 2015: https://groups.google.com/forum/#!topic/golang-nuts/e5qIlnhpgg0
The two points I note are:
1. when only one possible error condition, a bool is fine.
2. using an error is more flexible for future updates.

I think for Say, it meets no. 1 (out of range is the !ok case), and doesn't warrant no. 2.

Reply:
Thanks @leenipper that was sort of my take on it too. Good to see I was not alone in it!

Reply:
@leenipper thanks for catching that. New patch on the way soon.